zcash_client_sqlite/src/wallet.rs

```diff
@@ -218,6 +218,27 @@ impl Account {
             ViewingKey::Incoming(uivk) => uivk.default_address(request),
         }
     }
+
+    fn intersect_ua_request(
+        &self,
+        request: UnifiedAddressRequest,
+    ) -> Option<UnifiedAddressRequest> {
+        let uivk = self.viewing_key.uivk();
+
+        #[cfg(feature = "orchard")]
+        let has_orchard = uivk.orchard().is_some();
+        #[cfg(not(feature = "orchard"))]
+        let has_orchard = false;
+
+        let has_sapling = uivk.sapling().is_some();
+        let has_p2pkh = uivk.transparent().is_some();
+
```

Comment:
`Account` seems like the wrong place for this behavior. I think that a method `UnifiedIncomingViewingKey::to_address_request` would be better; it's not clear how this is intended to be used without going and looking at the call site.

Reply:
I wasn't confident about where this fn should go either. Happy to move it.

Reply:
I've removed it as you suggested it.